README.md

```diff
@@ -122,113 +122,115 @@ The devices below are fully feature supported by OpenRazer, which means all avai
+| Razer Naga X                                                  |  1532:0096  |
+| Razer DeathAdder Essential (2021)                             |  1532:0098  |
+| Razer Basilisk V3                                             |  1532:0099  |
+| Razer Pro Click Mini (Receiver)                               |  1532:009A  |
+| Razer DeathAdder V2 X HyperSpeed                              |  1532:009C  |
+| Razer Viper Mini Signature Edition (Wired)                    |  1532:009E  |
+| Razer Viper Mini Signature Edition (Wireless)                 |  1532:009F  |
+| Razer DeathAdder V2 Lite                                      |  1532:00A1  |
+| Razer Cobra                                                   |  1532:00A3  |
+| Razer Viper V2 Pro (Wired)                                    |  1532:00A5  |
+| Razer Viper V2 Pro (Wireless)                                 |  1532:00A6  |
+| Razer Naga V2 Pro (Wired)                                     |  1532:00A7  |
+| Razer Naga V2 Pro (Wireless)                                  |  1532:00A8  |
+| Razer Basilisk V3 Pro (Wired)                                 |  1532:00AA  |
+| Razer Basilisk V3 Pro (Wireless)                              |  1532:00AB  |
+| Razer Cobra Pro (Wired)                                       |  1532:00AF  |
+| Razer Cobra Pro (Wireless)                                    |  1532:00B0  |
+| Razer DeathAdder V3                                           |  1532:00B2  |
+| Razer HyperPolling Wireless Dongle                            |  1532:00B3  |
+| Razer Naga V2 HyperSpeed (Receiver)                           |  1532:00B4  |
+| Razer DeathAdder V3 Pro (Wired)                               |  1532:00B6  |
+| Razer DeathAdder V3 Pro (Wireless)                            |  1532:00B7  |
+| Razer Viper V3 HyperSpeed                                     |  1532:00B8  |
+| Razer Basilisk V3 X HyperSpeed                                |  1532:00B9  |
+| Razer Viper V3 Pro (Wired)                                    |  1532:00C0  |
+| Razer Viper V3 Pro (Wireless)                                 |  1532:00C1  |
+| Razer DeathAdder V3 Pro (Wired)                               |  1532:00C2  |
+| Razer DeathAdder V3 Pro (Wireless)                            |  1532:00C3  |
+| Razer DeathAdder V3 HyperSpeed (Wired)                        |  1532:00C4  |
+| Razer DeathAdder V3 HyperSpeed (Wireless)                     |  1532:00C5  |
+| Razer Pro Click V2 Vertical Edition (Wired)                   |  1532:00C7  |
+| Razer Pro Click V2 Vertical Edition (Wireless)                |  1532:00C8  |
+| Razer Basilisk V3 35K                                         |  1532:00CB  |
+| Razer Basilisk V3 Pro 35K (Wired)                             |  1532:00CC  |
+| Razer Basilisk V3 Pro 35K (Wireless)                          |  1532:00CD  |
+| Razer Basilisk V3 Pro 35K Phantom Green Edition (Wired)       |  1532:00D6  |
+| Razer Basilisk V3 Pro 35K Phantom Green Edition (Wireless)    |  1532:00D7  |
```

Comment:
This name is super long, is there no other special Basilisk V3 Pro 35K edition that might have the same USB PID? We've also had before that there's two basically identical mice with the same marketing name with different USB PIDs, so we could also just name it without the "Phantom Green Edition".

In any case, please separate this here into two changes - one to reformat the table to add more space for your name, and then the "Add support for [...]" change should only contain new two lines

Reply:
Ah, it might've been, I just looked at the "Files Changed" on GitHub which shows everything in one. If it's not done as I want, I'll fix this when merging. No worries then.

And thanks for the context regarding Logo LED.